ui/package.json

```diff
@@ -186,6 +187,7 @@
     "title-case": "^3.0.3"
   },
   "resolutions": {
-    "ember-auto-import": "^2.4.0"
+    "ember-auto-import": "^2.4.0",
+    "prop-types": "^15.8.1"
```

Comment:
this might be related:
* https://github.com/jkusa/ember-arg-types/issues/95
* https://github.com/jkusa/ember-cli-clipboard/issues/388

Reply:
this should fix it https://github.com/jkusa/ember-arg-types/pull/99